current/advanced/block-import.md

```diff
@@ -0,0 +1,90 @@
+---
+slug: block-import
+lang: en
+title: The Block Import Pipeline
+---
+
+## The Import Queue
+
+The import queue is an abstract worker queue present in every Substrate node. It is not part of the
+runtime. The Import Queue is responsible for processing pieces of incoming information, verifying
+them, and if they are valid, importing them into the node's state. The most fundamental piece of
+information that the import queue processes is blocks themselves, but it is also responsible for
+importing consensus-related messages such as Justifications and, in light clients, finality proofs.
+
+The import queue collects incoming blocks from the network and stores them in a pool. The blocks are
```

Comment:
What about non-block stuff like what you mention above?

Reply:
Works the same for all of them. I chose to focus on the blocks because they are most important (every chain has them) and I understand that the justifications and finality proofs may be refactored. Given those two things, do you think this should be revised?

Reply:
The docs for `ImportQueue` use the word "elements" to describe the things in the queue...maybe you can just use that same terminology here? Coming from the paragraph above this one, the first thing that came to mind when I read the first sentence of this paragraph is "okay, but what about the things that _aren't_ blocks?"